### PR TITLE
Add Gitsigns and Sneak support

### PR DESCRIPTION
**Pull request by jasonlong**
This adds support for [Gitsigns](https://github.com/lewis6991/gitsigns.nvim) and [Sneak](https://github.com/justinmk/vim-sneak).

#### Gitsigns (for current line blame)
Matches `Comment`

CleanShot 2022-05-27 at 09 47 54@2x

#### Sneak
Matches `Search`

CleanShot 2022-05-27 at 09 54 38@2x

### CODE REVIEW

File: colors/nord.vim [hunk 1]
```diff
@@ -582,13 +582,21 @@ call s:hi("GitGutterChange", s:nord13_gui, "", s:nord13_term, "", "", "")
 call s:hi("GitGutterChangeDelete", s:nord11_gui, "", s:nord11_term, "", "", "")
 call s:hi("GitGutterDelete", s:nord11_gui, "", s:nord11_term, "", "", "")
 
+" Gitsigns
```
Comment (jvoisin) [inline colors/nord.vim:585]: Please move this in the `if has('nvim')` block

**Review by jvoisin — CHANGES_REQUESTED** [review]

File: colors/nord.vim (hunk 1 above)
Comment (jvoisin) [reply colors/nord.vim]: In the one from line 562, no need to create a new condition :)

**Review by jvoisin — APPROVED** [review]

**Review by svengreb — APPROVED** [review]
Hi @jasonlong :wave:, thanks for your contribution :+1:

Looks good to me and consistent with other plugins/features, even though the support for Neovim specific plugins will be moved to an [upcoming "Nord Neovim" port][1] we can still add it for the time being 👍🏼

[1]: https://github.com/arcticicestudio/nord/issues/157#issuecomment-924312218

**Review by arcticicestudio — APPROVED** [review]

**Comment by jasonlong** [comment]
Very excited for the Neovim port! 🙌🏼

**Pull request merged**